File: abc/abc258/a.py
# https://atcoder.jp/contests/abc258/submissions/34692031

h = 21
n = int(input())
if n > 59:
  h += 1
  n -= 60
print(h, ":", sep="", end="")
if n < 10: print(0, end="")
print(n)

File: abc/abc258/a.cs
// https://atcoder.jp/contests/abc258/submissions/46508889

using System;

public class A
{
  static void Main()
  {
    int k = int.Parse(Console.ReadLine());
    int h = 21;
    int w = 0;

    h += k / 60;
    w = k % 60;

    Console.Write(string.Format($"{h:d2}:{w:d2}"));
  }
}

